apps/admin/src/pages/reports-page.css

```
@@ -573,3 +573,268 @@
+     min() of the two segment formulas picks whichever is smaller at any given width, which is
+     exactly the steep segment below the crossing point and the shallow one above it (verified:
+     the steep line is smaller for width < ~332px, the shallow one is smaller everywhere after) -
+     an `@container` condition on THIS same element instead of `min()` doesn't work: a container
+     cannot gate its own style based on its own size (the CSS spec blocks that as circular), so a
+     `@container (min-width: 332px) { .wallets-compare { ... } }` block here silently tested the
+     parent .at-card's width instead (it also has container-type) - confirmed by the segment
+     switching at the card's width crossing 332px, not this row's. min() sidesteps that
+     entirely since it's plain value computation, not a conditional style rule. */
+  --ring-size: clamp(48px, min(calc(-66px + 54.5cqi), calc(-6px + 36.5cqi)), 180px);
+  --ring-scale: calc(var(--ring-size) / 180px);
+  display: flex;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)